.github/workflows/notebooks-digest-updater-upstream.yaml

```diff
@@ -4,16 +4,21 @@ on:
     inputs:
       branch:
         required: true
-        description: "Provide the name of the branch you want to update ex main, vYYYYx etc"
+        description: "Provide branch name: "
       release-n:
         required: true
-        description: "Provide release N version of the notebooks ex 2023a"
-  schedule:
-    - cron: "0 0 * * 5" #Scheduled every Friday
+        description: "Provide Release N: "
+      release-n-1:
+        required: true
```

Comment:
seems like this required ?
does this mean, each time this would update both 2023a and 2023b ?

Reply:
I changed it and kept the versions as env variables. I think is better this way :+1: